src/app/modules/group/components/member-list/member-list.component.html

```diff
@@ -58,9 +58,9 @@
             </ng-container>
             <ng-container *ngSwitchCase="'name'">
               <a
-                  class="alg-link"
-                  [routerLink]="[ '/', 'groups', 'by-id', rowData.id, 'details' ]"
-                  >
+                class="alg-link"
+                [routerLink]="getSubGroupUrl(rowData.id)"
```

Comment:
Don't we avoid at all cost function calls in templates?  
Actually I know that 's not as dramatic as I thought...
Anyway, as the path is exactly the same for all children (I assume?), shouldn't you just do a pipe which build this path based on the id and parent info?

Reply:
Fixed